src/test/java/io/debezium/connector/vitess/VitessConnectorIT.java

```diff
@@ -333,23 +376,28 @@ private void waitForStreamingRunning() throws InterruptedException {
 
     private SourceRecord assertInsert(
                                       String statement,
-                                      List<SchemaAndValueField> expectedSchemaAndValuesByColumn) {
-        return assertInsert(statement, expectedSchemaAndValuesByColumn, TestHelper.TEST_UNSHARDED_KEYSPACE);
+                                      List<SchemaAndValueField> expectedSchemaAndValuesByColumn,
+                                      String pkField) {
+        return assertInsert(statement, expectedSchemaAndValuesByColumn, TestHelper.TEST_UNSHARDED_KEYSPACE, pkField, false);
     }
 
     private SourceRecord assertInsert(
                                       String statement,
                                       List<SchemaAndValueField> expectedSchemaAndValuesByColumn,
-                                      String database) {
+                                      String database,
+                                      String pkField,
+                                      boolean isMultiShardeds) {
```

Comment:
Should be "isMultiSharded"? Also worth a comment what it is.

Reply:
I renamed the function parameter to `hasMultipleShards` and added comments 👍